### PR TITLE
@dcuzzoni => Add some schemas for the show page

### PR DESCRIPTION
**Pull request by mzikherman**

### CODE REVIEW

**Comment by mzikherman** [comment]
I incorporated your comments from https://github.com/artsy/metaphysics/pull/39. Had to make a new fork/PR since the repo was made public when I had already forked the private one.

**Comment by mzikherman** [comment]
Maybe this can be merged for now? It's enough for the carousel and most of the header on the show page.

Gotta write some tests tho!

File: schema/fair.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    profile: {
+      type: Profile.type,
+      resolve: ({ default_profile_id, organizer }) => {
+        let id = default_profile_id || organizer && organizer.profile_id;
+        return gravity(`profile/${id}`)
+      }
+    },
+    has_full_feature: {
+      type: GraphQLBoolean,
+      resolve: ({ has_full_feature }) => has_full_feature
```
Comment (dzucconi) [inline schema/fair.js]: So you don't actually have to pull out these attributes with resolves, only resolve if you need to compute a value or there's something async

**Comment by mzikherman** [comment]
I think this is mergable!

I addressed all your comments, and it's getting pretty big already.

**Pull request merged**